snorkel/contrib/brat/tools.py

```diff
@@ -0,0 +1,392 @@
+import os
+import re
+import sys
+import glob
+import codecs
+from sqlalchemy.sql import select
+from collections import defaultdict
+from ...db_helpers import reload_annotator_labels
+from ...parser import TextDocPreprocessor, CorpusParser
+from ...models import Candidate, StableLabel, Document, TemporarySpan, Sentence, candidate_subclass
+
+
+class Brat(object):
```

Comment:
Skipping over this whole class for now... @jason-fries if you think this code is stable and you've been using it, feel free to merge in; otherwise make separate PR.

Either way, I think what this really needs--both to help me understand how to evaluate it and, more importantly, to help with actual users--is a short tutorial.  Can talk to @nishithbsk and/or the Arabidopsis team for help with this!